import.go

```diff
@@ -211,3 +333,24 @@ func (i *Importer) Commit() error {
 	i.Close()
 	return nil
 }
+
+// waitAndCloseChannels will try to close all the channels for importer and wait for remaining work to be done.
+// This function should only be called in the Commit or Close action. If any error happens when draining the remaining data in the channel,
+// The error will be popped out and returned.
+func (i *Importer) waitAndCloseChannels() error {
+	// Make sure all pending works are drained and close the channels in order
+	if !i.chanConfig.allChannelClosed {
+		i.chanConfig.allChannelClosed = true
+		close(i.chanConfig.chNodeData)
+		i.chanConfig.chNodeDataWg.Wait()
+		close(i.chanConfig.chBatch)
+		i.chanConfig.chBatchWg.Wait()
+		// Check errors
+		select {
+		case err := <-i.chanConfig.chError:
+			return err
+		default:
```

Comment:
Did you intend to add this default case? I ask because if an error wasn't already in the channel under multiplexing, this select default clause immediately executes and doesn't wait for anything on the channel.